main.go

```diff
@@ -146,6 +145,54 @@ func handle(w http.ResponseWriter, r *http.Request) {
 			http.Error(w, err.Error(), http.StatusInternalServerError)
 			return
 		} else if errors.Is(err, fs.ErrNotExist) {
+			srcDir, err := os.Getwd()
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
+				return
+			}
+
+			// determine package name to use for go get and go build
+			getPkg := ""
+			buildPkg := "."
+			buildDir := srcDir
+			if len(flag.Args()) > 0 {
+				pkg := flag.Args()[0]
+
+				if _, err := os.Stat(pkg); err != nil {
+					getPkg = pkg
+					buildPkg, _, _ = strings.Cut(pkg, "@")
```

Comment:
`Cut` is added from Go 1.18, so we would have to update go.mod. I'm a little against doing this. Would it be hard to fix this not to use `Cut`?

Reply:
Oh that wouldn't be hard, I wasn't aware that Cut was new. Let me fix that.

Reply:
there, latest commit should resolve